src/components/Identifier/Identifier/Identifier.stories.tsx

```diff
@@ -0,0 +1,426 @@
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="two">
+    <IdentifierLink href="#">Accessibility support</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="three">
+    <IdentifierLink href="#">FOIA requests</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="four">
+    <IdentifierLink href="#">No FEAR Act data</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="five">
+    <IdentifierLink href="#">Office of the Inspector General</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="six">
+    <IdentifierLink href="#">Performance reports</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="seven">
+    <IdentifierLink href="#">Privacy policy</IdentifierLink>
+  </IdentifierLinkItem>,
+]
+
+const testLinksSpanish = [
+  <IdentifierLinkItem key="one">
+    <IdentifierLink href="#">Acerca de &lt;Parent shortname&gt;</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="two">
+    <IdentifierLink href="#">Soporte de accesibilidad</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="three">
+    <IdentifierLink href="#">Solicitud a través de FOIA</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="four">
+    <IdentifierLink href="#">Datos de la ley No FEAR</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="five">
+    <IdentifierLink href="#">Oficina del Inspector General</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="six">
+    <IdentifierLink href="#">Informes de desempeño</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="seven">
+    <IdentifierLink href="#">Política de privacidad</IdentifierLink>
+  </IdentifierLinkItem>,
+]
+
+const testIdentifierGovContent = [
+  <div className="usa-identifier__container" key="one">
+    <div
+      data-testid="identifierGov-description"
+      className="usa-identifier__usagov-description">
+      Looking for U.S. government information and services?
+    </div>
+    &nbsp;
+    <a
+      data-testid="identifierGov-link"
+      href="https://www.usa.gov/"
+      className="usa-link">
+      Visit USA.gov
+    </a>
+  </div>,
+]
+
+const testIdentifierGovContentSpanish = [
+  <div className="usa-identifier__container" key="two">
+    <div
+      data-testid="identifierGov-description"
+      className="usa-identifier__usagov-description">
+      ¿Necesita información y servicios del Gobierno?
+    </div>
+    &nbsp;
+    <a
+      data-testid="identifierGov-link"
+      href="https://www.usa.gov/espanol/"
+      className="usa-link">
+      Visite USAGov en Español
+    </a>
+  </div>,
+]
+
+export const identifierDefault = (): React.ReactElement => (
+  <Identifier>
+    <IdentifierMasthead aria-label="Agency identifier">
+      <div className="usa-identifier__container">
+        <div className="usa-identifier__logos">
+          <IdentifierLogo
+            agencyUrl="#"
```

Comment:
Now that the logo is not necessarily connected to a specific agency, we can simplify `agencyUrl` to `href` to make it more like other "Linking" components, which should also make it more obvious what the component does besides displaying an image.